app/controllers/mappings_controller.rb

```diff
@@ -15,44 +15,49 @@ class MappingsController < ApplicationController
   INTERPORTAL_HASH = $INTERPORTAL_HASH ||= {}
 
   def index
-    ontology_list = LinkedData::Client::Models::Ontology.all.select { |o| !o.summaryOnly }
-    ontologies_mapping_count = LinkedData::Client::HTTP.get("#{MAPPINGS_URL}/statistics/ontologies")
-    ontologies_hash = {}
-    ontology_list.each do |ontology|
-      ontologies_hash[ontology.acronym] = ontology
+    @ontologies_mapping_count = LinkedData::Client::HTTP.get("#{MAPPINGS_URL}/statistics/ontologies")
+    ontologies = LinkedData::Client::Models::Ontology.all(
+      include: 'acronym,name,summaryOnly',
```

Comment:
why do need `summaryOnly` here  ?

Reply:
also, note that before they filtred out summaryOnly ontologies, you don't that anymore